### ISSUE TITLE
debug toolbar is invisible

### ISSUE THREAD

**Issue by ibell** [eranif/codelite#606]
I was having issues with codelite (another issue to follow), and so I thought I might try a re-install, but when i did so, I found that the debug toolbar was invisible, both in development and debug mode.  I toggled it many times, but I can't get it to show up.

**Comment by tongithub** [comment]
I have the same bug with version 7.0. Could it be, that the debug toolbar is simply out of the windows bounds by default? I dont have a larger screen available to check this, yet. Is there a way to adjust this in a config file?

**Comment by eranif** [comment]
I'm version 7, the toolbar is hidden by default.
did you try: view-> show toolbar?

**Comment by ksprwhite** [comment]
I have same issue. search, build, debug toolbars are invisibles.

![toolbar issue][1]


[1]: http://i.imgur.com/axgrn7y.png

**Comment by eranif** [comment]
Try this:
- Perspective->restore default layout
- Make sure that View->Show Toolbar is enabled

If the problem presists, enable the "Native toolbar" and restart codelite (again make sure that View->Show Toolbar is ON):
Settings->Global Editor preferences->Misc->Use native toolbar

**Comment by tongithub** [comment]
I found my Debug-ToolBar!
View-> show toolbar was selected. The Standard-, Search-, and Build-ToolBar was visible but the Debug-ToolBar was just not on screen. It was hidden behind the CodeLite window's right bounds. I was able to get it by using too large screen resolution settings from within a VM. Possibly it will also work if you use a virtual desktop, to make the CodeLite window bigger than the screen really is.
